### PR TITLE
Feat/next in q tests

### PR DESCRIPTION
**Pull request by mostarz93**

### CODE REVIEW

File: contracts/tests/LineLends.t.sol [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+import {LineOfCredit} from "../modules/credit/LineOfCredit.sol";
+
+import {Escrow} from "../modules/escrow/Escrow.sol";
+import {EscrowLib} from "../utils/EscrowLib.sol";
+import {IOracle} from "../interfaces/IOracle.sol";
+import {ILineOfCredit} from "../interfaces/ILineOfCredit.sol";
+import {RevenueToken} from "../mock/RevenueToken.sol";
+import {SimpleOracle} from "../mock/SimpleOracle.sol";
+
+contract LineLendsTest is Test {
```
Comment (kibagateaux) [inline contracts/tests/LineLends.t.sol:21]: lol u still didnt rename the file to "Lens"

File: contracts/modules/credit/LineOfCredit.sol [hunk 2]
```diff
@@ -544,8 +544,18 @@ contract LineOfCredit is ILineOfCredit, MutualConsent, ReentrancyGuard {
     }
 
     /// see ILineOfCredit.nextInQ
-    function nextInQ() external view returns (bytes32, address, address, uint256, uint256) {
+    function nextInQ() external view returns (bytes32, address, address, uint256, uint256, uint256, uint128, uint128) {
         bytes32 next = ids[0];
-        return (next, credits[next].lender, credits[next].token, credits[next].principal, credits[next].interestAccrued);
+        (uint128 drawnRate, uint128 facilityRate, ) = interestRate.rates(next);
+
```
Comment (kibagateaux) [inline contracts/modules/credit/LineOfCredit.sol:550]: still working on logic for returning null if no debt has been withdrawn right? I would expect a `if(position.principal == 0)` branch or something

**Review by kibagateaux — CHANGES_REQUESTED** [review]

File: contracts/modules/credit/LineOfCredit.sol (hunk 2 above)
Comment (mostarz93) [reply contracts/modules/credit/LineOfCredit.sol]: fixed

File: contracts/modules/credit/LineOfCredit.sol [hunk 3]
```diff
@@ -544,8 +544,30 @@ contract LineOfCredit is ILineOfCredit, MutualConsent, ReentrancyGuard {
     }
 
     /// see ILineOfCredit.nextInQ
-    function nextInQ() external view returns (bytes32, address, address, uint256, uint256) {
+    function nextInQ() external view returns (bytes32, address, address, uint256, uint256, uint256, uint128, uint128) {
         bytes32 next = ids[0];
-        return (next, credits[next].lender, credits[next].token, credits[next].principal, credits[next].interestAccrued);
+        (uint128 drawnRate, uint128 facilityRate, ) = interestRate.rates(next);
```
Comment (kibagateaux) [inline contracts/modules/credit/LineOfCredit.sol:549]: move logic to CreditLib so we save bytecode in SecuredLine contract

**Pull request merged**